week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,16 +7,25 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
 
+function journeyPlanner(londonLocations, transportMode){
+  let bus =[];
+    bus = londonLocations.filter(Element => Element.includes(transportMode) );
```

Comment:
Is `bus` the best name for this variable? How could we improve it?

Otherwise, this is looking great!